### PR TITLE
Add simple submit helper behaviour

### PR DESCRIPTION
**Pull request by sled**
Added a simple submit behaviour which allows to submit a sibling form etc.
Scenario is a "Save" button in the header which submits the form below.

### CODE REVIEW

File: app/assets/javascripts/mtl/submit.coffee [hunk 1]
```diff
@@ -0,0 +1,3 @@
+$(document).on 'click', '[data-mtl-submit]', (ev) ->
```
Comment (lwe) [inline app/assets/javascripts/mtl/submit.coffee]: Ao we require a preventDefault() here maybe, so when it's e.g. bound to a `<a>` tag?

**Comment by lwe** [comment]
Following inputs:

- Do you think it would make sense to have a clickables.coffee with both the mtl-href and mtl-submit in it?
- Also, please update REFERENCE.md with proper documentation, danke

**Comment by sled** [comment]
@lwe I combined `href.coffee` and `submit.coffee` into `clickables.coffee`. Also added a code example similar to `data-mtl-href` - please check, thx :)

File: app/assets/javascripts/mtl/clickables.coffee [hunk 2]
```diff
@@ -8,3 +8,8 @@ $(document).on 'click', '[data-mtl-href]', (ev) ->
 
 $(document).on 'click', '[data-mtl-href] a', (ev) ->
   ev.stopPropagation()
+
+$(document).on 'click', '[data-mtl-submit]', (ev) ->
+  selector = $(this).data('mtl-submit')
+  $(selector).submit()
+  ev.preventDefault()
```
Comment (lwe) [inline app/assets/javascripts/mtl/clickables.coffee]: nitpick, but reorder this please, e.g. do as the first thing, thx

Comment (sled) [inline app/assets/javascripts/mtl/clickables.coffee]: 👍 reorderd

**Pull request merged**